tools/accuracy_checker/accuracy_checker/annotation_converters/super_resolution_converter.py

```diff
@@ -43,6 +43,12 @@ def parameters(cls):
             'data_dir': PathField(
                 is_directory=True, description="Path to folder, where images in low and high resolution are located."
             ),
+            'lr_dir_suffix': StringField(
+                optional=True, description="Suffix of directory, where images in low resolution are located."
+            ),
+            'hr_dir_suffix': StringField(
+                optional=True, description="Suffix of directory, where images in high resolution are located."
+            ),
```

Comment:
it is not present in the original dataset but used in some cases, please provide also opportunity to set directory for upsample images